### DISCUSSION TITLE
Editable columns for the devies, entities and areas lists.

### DISCUSSION THREAD

**Discussion by 10bn** [home-assistant/frontend#8685] [Accessibility]
Examples for Devices / Entities / Areas Management: 

![image](https://user-images.githubusercontent.com/46263124/111902082-d79c9980-8a3b-11eb-8068-618569883f2b.png)
![2021-03-22 09_24_30-Configuration - Home Assistant](https://user-images.githubusercontent.com/46263124/111961209-0e80b700-8af1-11eb-902f-3680d08bec5a.jpg)

It would super handy if we could edit those deep down in the settings hidden names or affiliations directly in the appropriate list / table.

**Comment by spacegaier** [comment]
I don't see the benefit of this. Currently it is one click on the entity row and you are in the dialog where you can change the name and the entity ID.

In your mock-up you have a drop down box, but it would need to be an input field?

**Comment by 10bn** [comment]
sry, the drop down is wrong I'll update the picture. 

You are right, it's one klick into the settings and another click into the sub settings, if want to change just one entry it's fine, but
especially when you are restructuring or working with a fresh setup it's a pain to rename many entities or assigning areas.
